p2p/net/mock/mock_test.go

```diff
@@ -368,7 +374,7 @@ func TestStreamsStress(t *testing.T) {
 	ctx := context.Background()
 	nnodes := 100
 	if detectrace.WithRace() {
-		nnodes = 50
+		nnodes = 30
```

Comment:
This is OK (and even better) because we're stress testing _streams_, so reducing the host count increases stream density per host.